### DISCUSSION TITLE
Your Canarytoken was Triggered - Email change this week (24/07/24)

### DISCUSSION THREAD

**Discussion by wilmerism** [thinkst/canarytokens#550] [Q&A] [answered]
Morning, 

Monday me email alertrs changed cosmetically. IT looks nice but is now missing the "basic details" information. I use the same tokens on multiple devices so having "Executed on" and "Run by"  information on the email body was great. I knew straight away what & where (even on a mobile phone) 

Now with this new layout I have to look at "Alert History"

I dont want to knock it as this is the free one im using. but any reason for less info / any plans for return of information to the mail body ?

**Comment by ranok** [comment]
Thank you @wilmerism for raising this. We are in process of the larger update/modernization and adding back these extra pieces of information to the mail alerts is underway right now. Sorry that you didn't get all the info upfront, but in the next few days it should be fixed and deployed so your future alerts will have that information within.

Thanks for your patience and for being a user!

**Reply by ranok** [reply, marked as answer]
This has been pushed to prod and should now be resolved. Thanks again @wilmerism
